config_src/external/smartredis/smartredis_client.F90

```diff
@@ -0,0 +1,925 @@
+module smartredis_client
+
```

Comment:
Needs the MOM6 license one-liner.  This is the stub which lives in MOM6 so must hae the MOM6 license. The actual code that replaces the stub can use whatever license you like.